Makefile.work

```diff
@@ -527,7 +530,8 @@ endif
 
 	$(Q)$(DOCKER_RUN) \
 		$(SLAVE_IMAGE):$(SLAVE_TAG) \
-		bash -c "$(SONIC_BUILD_INSTRUCTION) $@;$(COLLECT_BUILD_VERSION); $(SLAVE_SHELL)"
+		bash -c "$(SONIC_BUILD_INSTRUCTION) $@ && $(COLLECT_BUILD_VERSION) && $(SLAVE_SHELL)" \
```

Comment:
This could be problematic. Currently, `$(COLLECT_BUILD_VERSION)` is meant to be run always (whether the actual build passed or not). That script then exits with the exit code that's passed in.

Reply:
done